plugin/util/parse_bindings.go

```diff
@@ -74,7 +74,12 @@ func parseBindingObjList(topList *ast.ObjectList) (map[string]StringSet, error)
 			continue
 		}
 
-		key := item.Keys[0].Token.Value().(string)
+		keyRaw := item.Keys[0].Token.Value()
```

Comment:
Probably we also need a nil check on keyRaw -- since it's an interface, it *could* be nil.